packages/plugin-preset/src/suggestors/packages-with-extra-artifacts.ts

```diff
@@ -40,6 +40,7 @@ export class PackagesWithExtraArtifacts extends SuggestionTask {
               size.physical
             )}.`,
             targetPackageId: `${node.name}@${node.version}`,
+            priority: size.physical,
```

Comment:
make this an enum / utility that will allow developers to have 'LOW' | 'MEDIUM' | 'HIGH'

also add a impactScore that is a 0 - 100.